vmhost/hostCore/execution.go

```diff
@@ -332,6 +333,14 @@ func (host *vmHost) ExecuteOnDestContext(input *vmcommon.ContractCallInput) (vmO
 			isChildComplete = true
 			return
 		}
+		switch input.CallType {
+		case vm.AsynchronousCall:
+			host.CompleteLogEntriesWithCallType(vmOutput, "AsyncCall")
+		case vm.AsynchronousCallBack:
+			host.CompleteLogEntriesWithCallType(vmOutput, "AyncCallback")
+		default:
+			host.CompleteLogEntriesWithCallType(vmOutput, "ExecuteOnDestContext")
+		}
 	}
 
 	isChildComplete = true
```

Comment:
do not create transferValueOnly log event if you already created one on lines 336-343